core/src/main/codegen/templates/Parser.jj

```diff
@@ -1518,11 +1521,18 @@ SqlNode SqlDescribe() :
 {
     <DESCRIBE> { s = span(); }
     (
-        LOOKAHEAD(2) (<DATABASE> | <CATALOG> | <SCHEMA>)
+        <DATABASE>
```

Comment:
What is the plan next step?  Can resolve the issue by add Describe DABASE/CATALOG/STATEMENT to server module parserImpls.ftl which would generate code in SqlDdlParserImpl.